frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1298,6 +1368,14 @@ function ReaderHighlight:onReadSettings(config)
         disable_highlight = G_reader_settings:readSetting("highlight_disabled") or false
     end
     self.view.highlight.disabled = disable_highlight
+
+    -- panel zoom settings shouldn't work in EPUB
```

Comment:
I'd phrase it slightly differently because if I look at my comics EPUB is a distinct download option! ;-D
```suggestion
    -- panel zoom settings isn't supported in EPUB
```

![Screenshot_2020-09-24_15-15-05](https://user-images.githubusercontent.com/202757/94149888-c3a32100-fe78-11ea-9c94-963204ace53a.png)